packages/sonarwhal/src/lib/utils/misc.ts

```diff
@@ -186,6 +186,17 @@ const toCamelCase = (value: string) => {
     }, '');
 };
 
+/** Convert '-' delimitered string to pascal case name. */
+const toPascalCase = (value: string) => {
```

Comment:
You are adding this method that doesn't seem to be used anywhere else in this PR.

Reply:
This is too weird, I should be using this method in new-rule.ts

Reply:
I'm using it in the file `new-rule.ts` line 102